srv/salt/ceph/igw/config/create_iscsi_config.sls

```diff
@@ -14,7 +14,40 @@
         trusted_ip_list: {{ ip_addresses }}
 {% endfor %}
 
+{% if pillar.get('ceph_iscsi_ssl', False) %}
+{% if pillar.get('ceph_iscsi_ssl_cert', None) and pillar.get('ceph_iscsi_ssl_key', None) %}
+
+/srv/salt/ceph/igw/cache/tls/certs/iscsi-gateway.crt:
+  file.managed:
+    - source: {{ pillar['ceph_iscsi_ssl_cert'] }}
+    - template: jinja
```

Comment:
This probably shouldn't run through a jinja interpreter? Same lin line 33

Reply:
you're right, copy&paste leftover. Will change it.

Reply:
sorry for the delay, change is done.